README.md

```diff
@@ -81,6 +81,7 @@ script.Args().Concat().Match("Error").First(10).AppendFile("/var/log/errors.txt"
 		- [Error output](#error-output)
 	- [File](#file)
 	- [IfExists](#ifexists)
+	- [FindFiles](#findfiles)
```

Comment:
Shall we add this to the list of Unix equivalents, as well, as the equivalent of `find`?

Reply:
Yea, I think so, although a bit limited as we don't have glob support for find file.